docs/README.md

```diff
@@ -54,6 +54,40 @@ kubectl apply -f secret.yaml
 
 * Using worker node instance profile - grant all the worker nodes with proper permission by attach policy to the instance profile of the worker.
 
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iam:CreateServiceLinkedRole",
+        "iam:AttachRolePolicy",
+        "iam:PutRolePolicy"
+       ],
+      "Resource": "arn:aws:iam::*:role/aws-service-role/s3.data-source.lustre.fsx.amazonaws.com/*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "fsx:*"
+      ],
+      "Resource": ["*"]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
```

Comment:
I think `s3:*` and `ec2:CreateNetworkInterface` is not needed by driver itself

Reply:
I see. I remembered I get some Access Denied error. Let me double check on my end.

Reply:
@leakingtapan Make the change and remove unnecessary permissions. Sorry for late update